calendar-webapp/src/main/java/org/exoplatform/calendar/webui/UICalendarPortlet.java

```
@@ -454,4 +451,18 @@ public void createActionConfirms(UIConfirmation uiConfirmation) {
     actionConfirms.add(new UIConfirmation.ActionConfirm("AbortClose", no));
     uiConfirmation.setActions(actionConfirms);
   }
+
+  /**
+   * use in template
+   */
+  public int getTimeShift(){
+    int defaultTime = 2;
+    try {
+      defaultTime = Integer.parseInt(PropertyManager.getProperty("exo.calendar.default.event.suggest"));
+    } catch (NumberFormatException e) {
+      defaultTime = 1;

```

We should not swallow the exception and log an error to warn the administrator that the configuration is not correct.

---

ok let me fix this

---

@thomasdelhomenie , i'm done fixing log and increase test coverage rate